examples/knative-eventing-tutorial/Dockerfile

```
@@ -0,0 +1,18 @@
+FROM golang:1-buster AS builder
+WORKDIR /app
+RUN apt-get update -qq && apt-get install -qqy protobuf-compiler
+RUN go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.26
+RUN go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.1
```

RUN commands must be chained with `&& \` (please fix the indent)
```suggestion
RUN apt-get update -qq && apt-get install -qqy protobuf-compiler && \
 go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.26 && \
 go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.1
```

---

I will generate the protobuf stubs on host and pass it to the Docker then (given that it's a mostly deterministic process).